src/bin/nydus-image/main.rs

```
@@ -1734,6 +1819,32 @@ impl Command {
         }
     }
 
+    fn get_blobs_dir(matches: &ArgMatches) -> Result<&Path> {
+        match matches.get_one::<String>("blob-dir") {
+            Some(s) => Ok(Path::new(s)),
+            None => bail!("missing parameter `blob-dir`"),
+        }
+    }
+
+    fn get_prefetch_files(matches: &ArgMatches) -> Result<Vec<String>> {
+        match matches.get_one::<String>("prefetch-files") {
+            Some(v) => {
+                let content = std::fs::read_to_string(v)
+                    .map_err(|_| anyhow!("failed to read prefetch files from {}", v))?;
```

Do not ignore the err.